src/aequitas/flow/experiment/experiment.py

```diff
@@ -258,3 +260,14 @@ def dt_handler(x):
             json.dumps(self.config, default=dt_handler, sort_keys=True).encode("utf-8")
         ).hexdigest()
         self.logger.debug(f"Hash generated: {self.hash}.")
+
+    def plot_pareto(
+        self,
+        dataset: str,
```

Comment:
Can we have a default `"Dataset"` here? It connects nicely to the default dataset name in `DefaultExperiment` (that's a lot of defaults).